pandas/tests/series/test_timezones.py

```diff
@@ -364,3 +364,11 @@ def test_tz_localize_convert_copy_inplace_mutate(self, copy, method, tz):
                           index=date_range('20131027', periods=5, freq='1H',
                                            tz=tz))
         tm.assert_series_equal(result, expected)
+
+    def test_series_set_tz_timestamp(self):
```

Comment:
yeah pls move to pandas/tests/indexing/test_scalar.py, near this one: def test_at_with_tz

Reply:
I've moved the test there